### ISSUE TITLE
[FeatureRequest] Firmware update notifications

### ISSUE THREAD

**Issue by andreas-bulling** [Tinkerforge/warp-charger#42]
Compared to many other systems I know (e.g. HomeAssistant, OpenDTU, ESPHome, QNAP, Nextcloud) it's currently very difficult and tedious to find out about new firmware versions for the Charger and EM. Sure, there is a forum thread that one can frequently poll and the product pages also show the latest version. But this requires frequent and active initiative of the users.

I thus suggest, e.g. similar to OpenDTU, for the web interfaces of the Charger and the EM to offer a firmware check / update notification that informs users about new firmware versions. Ideally by also offering a button to directly download and install the latest version.

In parallel it would also be good to have the latest firmwares as "Releases" on GitHub - again, to make it easier to find them (where people would also expect them to be). The update notification / firmware check could directly get its information from there.

**Comment by photron** [comment]
Still a duplicate of https://github.com/Tinkerforge/esp32-firmware/issues/88.
